hypervisor/qemu/qemu_arm64.go

```diff
@@ -42,7 +42,7 @@ func (qc *QemuContext) arguments(ctx *hypervisor.VmContext) []string {
 		cpuParams = strconv.Itoa(boot.CPU)
 	}
 
-	params := []string{"-machine", "virt,accel=kvm,usb=off", "-global", "kvm-pit.lost_tick_policy=discard", "-cpu", "cortex-a57"}
+	params := []string{"-machine", "virt,accel=kvm,gic-version=3,usb=off", "-global", "kvm-pit.lost_tick_policy=discard", "-cpu", "host"}
```

Comment:
GICv3 is the ARM generic interrupt controller designed to overcome
some limits of the prevalent GICv2. Most notably it lifts the 8-CPU
limit. Though with Linux-3.17 Marc introduced support for hosts to
use a GICv3, the CPU limitation still applies to KVM guests, since
the current code emulates a GICv2 only.
Also, GICv2 backward compatibility being optional in GICv3, a number
of systems won't be able to run GICv2 guests.

https://wiki.ubuntu.com/ARM64/QEMU
Note: For GICv3 systems, such as Cavium ThunderX, you must use QEMU from Ubuntu 16.04 or newer, and pass "-M virt,gic_version=3". Additionally, you must use a 15.10 (wily) or newer cloud image for guests.